src/Config_option.ml

```diff
@@ -9,6 +9,26 @@
  *                                                                    *
  **********************************************************************)
 
+module Removed_value = struct
```

Comment:
What do you think of moving this module inside `Make` ? So users can access it with same paths as other things of this module.

Reply:
Functors bodies tend to attract a lot of things, which makes them become harder and harder to understand as the code base grows. I was actually thinking to move this to its own file to keep the interface clear, what do you think about that?

Reply:
Maybe that would make a lot of files for a single concern ?
I don't think growing the body of functors is a bad thing, why would it be worse than growing a simple module ?
I was suggesting that so the usage is consistent, `C.choice` vs `Config_option.Removed_value.make`. `C.Removed_value` looks better to me because it's obvious from where it come.

Reply:
Removing the functor would be better indeed :)